### PR TITLE
[azdatalake] HNS encryption scope

### PR DESCRIPTION
**Pull request by tanyasethi-msft**
- [x] The purpose of this PR is explained in this or a referenced issue.
- [x] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [x] Tests are included and/or updated for code changes.
- [x] Updates to module CHANGELOG.md are included.
- [x] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go

### CODE REVIEW

File: sdk/storage/azdatalake/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 
+## 1.1.0-beta.2 (Unreleased)
+
+### Features Added
+* HNS Encryption Scope support
+
+### Breaking Changes
+
+### Bugs Fixed
+* Escape paths for NewDirectoryClient and NewFileClient in a file system. Fixes [#22281](https://github.com/Azure/azure-sdk-for-go/issues/22281).
```
Comment (vibhansa-msft) [inline sdk/storage/azdatalake/CHANGELOG.md:11]: Is this related to this changeset?

**Review by vibhansa-msft — APPROVED** [review]

**Review by vibhansa-msft — APPROVED** [review]

**Review by souravgupta-msft — APPROVED** [review]

File: sdk/storage/azdatalake/CHANGELOG.md (hunk 1 above)
Comment (tanyasethi-msft) [reply sdk/storage/azdatalake/CHANGELOG.md]: no, removed
